### ISSUE TITLE
Fix S2933 FP: Support unsigned right-shift operator (>>>) 

### ISSUE THREAD

**Issue by csaba-sagi-sonarsource** [SonarSource/sonar-dotnet#6313]
### Description

FP for [S2933](https://sonarsource.github.io/rspec/#/rspec/S2933/csharp) (FieldShouldBeReadonly).

Using C#11 unsigned right-shift operator (>>>), the rule is raising an issue.

### Repro steps

```
class Person
{
    int somefield = 42; // Noncompliant FP (it is part of an UnsignedRightShiftAssignmentExpression)
    private readonly Action<int> setter;

    Person(int birthYear)
    {
        setter = i => { somefield >>>= i; };
    }
}
```

### Expected behavior

The rule should not raise an issue.

### Actual behavior

The rule raises an issue.
